src/char.cr

```diff
@@ -69,6 +69,38 @@ struct Char
     ord - other.ord
   end
 
+  # Makes a new string by adding *str* to itself *times* times.
+  #
+  # ```
+  # '-' * 10 # => "----------"
+  # ```
+  def *(times count : Int)
```

Comment:
Sorry for joining late :-), but it's hard to imagine someone will write `"aString".*(times=3)`.
I am ok with either name, but I see no point with the named argument in an infix operator.

Unless there is a reason I would drop that. Again any internal name is ok.